recurly/__init__.py

```diff
@@ -467,6 +467,7 @@ class Invoice(Resource):
         'terms_and_conditions',
         'customer_notes',
         'address',
+        'closed_at',
```

Comment:
I decided to keep with the current pattern and include it. I'll gladly remove it though. FWIW most every attribute array in this file has a trailing `,`